packages/components/src/components.d.ts

```diff
@@ -128,6 +135,7 @@ declare global {
     'stencila-actionmenu': HTMLStencilaActionmenuElement;
     'stencila-button': HTMLStencilaButtonElement;
     'stencila-codechunk': HTMLStencilaCodechunkElement;
+    'stencila-codeexpression': HTMLStencilaCodeexpressionElement;
     'stencila-tab': HTMLStencilaTabElement;
```

Comment:
To maintain consistency in naming of schema types across code bases, I suggest that we change the naming of the custom elements to:

```js
    'stencila-code-chunk': HTMLStencilaCodeChunkElement;
    'stencila-code-expression': HTMLStencilaCodeExpressionElement;
```

Will require changes in Encoda now, but less painful in the long run than having inconsistent naming conventions.